### ISSUE TITLE
Some questions about the environment

### ISSUE THREAD

**Issue by zhanghuzhenyu** [haosulab/ManiSkill-Learn#27]
Thanks for the pretrained models. I found that these pretrained models were obtained on OpenCabinetDoor-v0,PushChair-v0,MoveBucket-v0 and OpenCabinetDrawer-v0. There are several different cabinet doors in OpenCabinetDoor-v0, are the pretrained models obtained on different doors,or on just one door?

What is the relationship between OpenCabinetDoor-v0 and OpenCabinetDoor_1000-v0? Does OpenCabinetDoor-v0 include OpenCabinetDoor_1000-v0?

And what is the relationship between OpenCabinetDoor_1000-v0 and OpenCabinetDoor_1000_link_0-v0?

In the article you say "For each task, the average test success rates are calculated over the 10 test environments and 50 evaluation
trajectories per environment.” If I want to get the results of the test by myself, which ten environments do I need to evaluate?

**Comment by xuanlinli17** [comment]
OpenCabinetDoor-v0 includes OpenCabinetDoor_1000-v0. OpenCabinetDoor_1000-v0 is an environment that randomizes the target drawer (link) of cabinet 1000 upon each env reset, i.e. it contains OpenCabinetDoor_1000_link_0-v0. Note that a single cabinet can contain multiple drawers / doors. OpenCabinetDoor_1000_link_0-v0 restricts the target drawer of cabinet 1000 to be the one corresponding to link 0.

Previously, during the ManiSkill challenge, results on the test objects are obtained by submitting to our evaluation server [(link)](https://sapien.ucsd.edu/challenges/maniskill2021/).  Since the ManiSkill Challenge has already finished, we are yet to devise the details about post-challenge evaluation on the test objects (such as frequency of evaluation). However, we do not currently plan to release the test objects and their models (though the evaluation results on the test objects will be available if you submit to the server later)

If you wish, you are welcome to create a new setting on your own, such as a manual train-validation split. You can do this by registering a new env following the examples in https://github.com/haosulab/ManiSkill/blob/main/mani_skill/env/__init__.py#L110-L112

**Comment by tongzhoumu** [comment]
@zhanghuzhenyu 

The test environments we used in the paper include the test objects which are not released to the public. Previously, you can evaluate your model via [ManiSkill Challenge](https://sapien.ucsd.edu/challenges/maniskill2021/challenges). However, this challenge was just concluded recently and we have closed the submission portal.

We are currently working on a new evaluation system and it will be opened to the public again ASAP. We will let you know when it is launched.

**Comment by zhanghuzhenyu** [comment]
I see, thanks a lot for your answer!

**Comment by tongzhoumu** [comment]
@zhanghuzhenyu 

Hi, we just launched a [temporary test server](https://sapien.ucsd.edu/challenges/maniskill2021test/) to provide a test set evaluation service. Feel free to use it.

**Comment by zhanghuzhenyu** [comment]
Thanks so much!!!
